src/server/scripts/Spells/spell_monk.cpp

```diff
@@ -159,9 +159,62 @@ class spell_monk_provoke : public SpellScript
     }
 };
 
+// Roll - 109132
+class spell_monk_roll : public SpellScriptLoader
+{
+public:
+    spell_monk_roll() : SpellScriptLoader("spell_monk_roll") { }
+
+    class spell_monk_roll_SpellScript : public SpellScript
+    {
+        PrepareSpellScript(spell_monk_roll_SpellScript);
+
+        bool Validate(SpellInfo const* /*spell*/) override
+        {
+            if (!sSpellMgr->GetSpellInfo(109132, DIFFICULTY_NONE))
```

Comment:
You should use `ValidateSpellInfo()` rather than `sSpellMgr->GetspellInfo()`.
Also, you're using spell id 107427, so the same spell id should be here as well.